### DISCUSSION TITLE
Captain - Few  suggestions of how to make the captain role realy recognised

### DISCUSSION THREAD

**Discussion by Swato85** [FakeFishGames/Barotrauma#7439] [Ideas and suggestions]
Hello here is a short list of possible improvements to captain role, so that it is really recognised for being  the "captain = authority" on the vessel.

1. Give all (literaly all) crew who follow captain orders somekind of bonus. Preferably the bonus should directly reflect the ordered task (request repairs-faster repairs buff, request medic service, decrease  medicine debuff chance, increase  pottency, whatever; request threat management-give damage bonus.
 - The important thing: link the buff with a debuff if the crew is not doing what captain ordered and there are options to do it.  Like for example dont debuf  a medic for not healing when it is not required, or mechanic for not repairing when there is nothing to repair and buff them instead for good maintenance. But definitely debuff  crew members  who not follow orders. 
 - special cases can be for if the ship is heavily damaged/sinking (and other issues), at that  time no debuffs  should be active.
Additionaly  - crew can get something like  a minimission  from these orders (e.g.:  oder for repairs, if 10 repairs are done, this crewmate receives somekind of  reward,  money/stat bonus/or some good buff untill the  end of round  and as long as he  keeps following orders.

2. Captain should have the option to order to detain/pacify someone. There security can deal with griefers/suspects and whoever in accordance to protocol and without negative impact from the game or other  crew as it will be a proper procedure.
  - closely linked to this is  a frisk search/inspection order, so that security can search  someone for ilegal items. Dont mix this with the above option as this does not allow to attack the suspect. You can implement this by setting somekind of timer untill when the target must ragdoll and allow security to search him - if this is not done in a predefined time, the  security will be allowed to use force  without negative impact.

3.  Captain may be closely linked to sub - to increase crews awareness that captain is  required for safe reach of the  destination.  If something happens to captain, the ship  should be worse controlable irelevant of helm, or suffer more collision/attack dmg, or equipment on board can occasionaly break and similar. This would reflect the theory that the  ship effectiveness is tighedtothe captain.

4. In theory the  vessel captain should be  recognised by outposts, so there can be some events linked to captain. e.g.:  Crew without captain can not dock immediately, but must wait for few minutes (docking permission being  discussed at the outpost).
 - buying a new sub and certain mission will notbe allowed unless the sub has a captain, or most things can cost more.

5. Link captain  to reputation. So if captain dies, the  reputation decreases (not  into negatives, just by some point  but to max of 0). This is  simply linked to that the vessel captains are usualy those to whom  is the reputation associated, so if the old captain died, there is some mistrust to the  new management  from the future  clients (something  similar like  to when master chef  leaves/dies a  the restaurant looses prestige).

6. Captain should be able to add/remove tags of ID cards in order to grant/remove access of  crew depending  on their  role and behaviour.

I believe these  few aspects could make a good impact on roleplay, so hopefully something gets atttention.

**Comment by ghost** [comment]
well your first point: There's already talents for that
point 2: Give your security orders in chat.
point 3: That'd make no sense, and also piloting with 0 captain is already hellish.
point 4: What does that do other than waste time?
point 5: Ok so captains are encouraged EVEN MORE to just hide in their ivory command room.
point 6: How would that work? and how would the captain know what the sub's tags are? what if the captain is new and doesn't know how to manage ID cards?

**Reply by Swato85** [reply]
1. No there is not, the  one in adresses only one  character, ussualy a friend of captains  in  MP.
2. Does not have any incentive to follow up, unless you play with good roleplayer guys. 
3. It was just a suggestion, it can be implemented in many different ways, the point was to somehow install in the  game the common sense as on real  boats. You  know the situations: "captain is missing, repply. Oh *, this  is not a good sign.
4. Yeah, just a possible suggestion on basis of real world unfortunately.
5. Well the realisation  is of  course that the captain sees what tags are where on HIS  boat, I  still remember years back  when I  bought  a ship and could not understand why I can not enter secure lockers and some doors :-) Captain should be aware of this and have control over it dont you think?
- an  Idea just camed to mind that this could be done even with  some item, like you have wrench and screwdriver and  captain can have somekind of tool or even better if his card was the tool, so the capability will be nicely locked to captain and  still being obtainable by traitors, saboteurs or simply other crew members. Thanks for the inspiratiion.

**Reply by ghost** [reply]
1. It can address 2 crew-members. If your crew is being disobedient you should be taking more measures to ensure cooperation. Like a gun.
2. No incentive to follow up? have you heard of a gun? Also if you're hosting a server and security are disobeying orders you have a kick button.
3.  On real boats, the usefulness of the captain is his orders and piloting I believe. Both are tasks that can be performed by anyone with enough training.

**Comment by Swato85** [comment]
On a side note, Captain should definitely get a perk to allow him to find extra physicorium or he mostly ends up with no  ammo for the gun. something like to the opening of loot in wrecks/ruins or a side find on monsters/minig, whatever. just so that his gun can beused more than just a decoration due to lack of ammo sources.

**Reply by ghost** [reply]
Not sure you realise but captain is not a super powerful role and really should not be. Plenty of other classes have many ways to get more alien materials, and a captain should be ordering their crew to produce or loot for more. It's also worth noting that the handcannon is extremely powerful, one-shotting humans with the right perks, so it should not be made cheap to use.

**Reply by Swato85** [reply]
It is not about making him super-strong, just giving him  various useability  just like the rest  of the crew.  This is not a mega ship, but a team of marines working  together, and also you can not ignore players who like to play together - do things together, so why not give them something to tag along.

**Comment by gerakis** [comment]
I actually want the "captain" (the guy who drives the boat) to be less in command than he is. Like if you want to be in charge of a ship on a server but you dont want to play as the captain that should be possible. Right now people would think "but you arent the captain, why should I listen to you."

I would say the Captain should be some extra title that can be awarded to any class of character, and replace "captain" class with "helmsman." That way the guy driving the boat isn't necessarily the guy in charge of the boat.

Like Picard in Star Trek. He was in charge, but he didnt drive the ship. He had a helmsman for that.

**Reply by Swato85** [reply]
Yep that is exactly the issue. Captain role is not exactly a captain role and should be reflected on that. You mentioned Picard, he is a good example of captain that realy Leads by byeing  an  example - now if you remember his regular away missions, try to mention that to local extremers, torches would rise just from giving  them the image of captain leaving his ship...  :-D. Some simply can not live  with that idea. Now waiting for the incoming  posts....
